packages/api/tests/models/test-granules-model.js

```diff
@@ -702,3 +700,36 @@ test(
     t.is(await granuleModel.createGranulesFromSns(cumulusMessage), null);
   }
 );
+
+test(
+  'reingest pushes a message with the correct queueName',
+  async (t) => {
+    const { granuleModel } = t.context;
+    granuleModel.updateStatus = () => {};
```

Comment:
This could interfere with other tests if any of them called `updateStatus` on the granule model, correct? It's probably best to stub this and restore, unless I'm missing something